cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -308,8 +306,6 @@ features:
     Header:
       filterBy: Filter by
       search-placeholder: Search
-      sortdropdown-tooltip: Clear Search to enable Sort
```

Comment:
not needed? this, and the next?

Reply:
Yep, couldn't find any references to these